test/src/app/App.test.js

```diff
@@ -137,6 +137,19 @@ contract('App', function ([_, owner]) {
         });
       });
 
+      describe('with complex initializer', function () {
+        beforeEach('creating a proxy', async function () {
+          this.proxy = await this.app.createProxy(ImplV1, contractName, 'initialize', [10, "foo", []]);
+        });
+
+        shouldReturnProxy();
+
+        it('should have initialized the proxy', async function () {
+          (await this.proxy.value()).toNumber().should.eq(10);
+          (await this.proxy.text()).should.eq("foo");
```

Comment:
Missing `(await this.proxy.values()).should.eq([])`?

Reply:
Public getters for array only work with indices, so you'd need to query `proxy.values(index)`. However, it's empty, so any call would fail. I can add that to the test though.